LICENSE.MIT

```diff
@@ -0,0 +1,30 @@
+------------------------------------------------------------------------------
+This license applies to the following third-party libraries included
+in this repository:
+
+  - backbone.paginator
+  - Backbone.js
+  - Underscore.js
```

Comment:
@itsjeyd Is it necessary to include these? Backbone and Underscore should both be available in Studio and the LMS:

```
RequireJS.require(['backbone', 'underscore'], function(backbone, _) {
    console.log(backbone.VERSION, _.VERSION);
});
> 1.0.0, 1.4.4
```